client/src/App.js

```diff
@@ -1,20 +1,226 @@
-import React from "react";
-import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
+import './App.css';
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Divider from '@material-ui/core/Divider';
+import InputLabel from '@material-ui/core/InputLabel';
+import Link from '@material-ui/core/Link';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import Typography from '@material-ui/core/Typography';
 
-import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+const useStyles = makeStyles({
+	buttons: {
+		fontFamily: "Trebuchet MS, sans-serif",
```

Comment:
In order to avoid repeating the font nam and colors in the app, you can add those to `theme.js` and reuse anywhere else. Here is some info on theming https://material-ui.com/customization/theming/

Reply:
I also added a style guide to our google drive that you can refer to for the exact color and font